src/components/UserProfileCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="profile_card" @click="$router.push('/login')" v-if="$store.state.currentUser">
+    <img :src="profilePicture" />
+    <div class="detail">
+      <div>{{$store.state.currentUser.displayName}}</div>
+    </div>
+  </div>
+</template>
+
+<script>
+import md5 from 'js-md5'
+export default {
+    name:"UserProfileCard",
+    computed: {
+        profilePicture(){
+            let hash = md5(this.$store.state.currentUser.email);
+            let gravatar_link = 'http://www.gravatar.com/avatar/' + hash + '?s=50';
+            return gravatar_link
```

Comment:
Missing semicolon? Maybe we need to add a lint rule for this (to avoid stupid issues like automatic semicolon insertion).

```suggestion
            return gravatar_link;
```

Reply:
Yes, for some reason es-lint isn't working in *.vue files, need to look into that, as the `vue-cli` will do linting in vue files as well, there might be a missed configuration.